lib/plugins/track.js

```diff
@@ -109,6 +109,15 @@ function configureStorage (storage) {
   }, storage)
 }
 
+function removeCookie (storage) {
+  var name = storage.name
+  var path = storage.path || '/'
+  var domainChunks = storage.domain.split('.')
+  for (var domain, i = domainChunks.length - 1; (domain = domainChunks.slice(i).join('.'), i >= 0); i--) {
```

Comment:
Nit: Since we use this looping thru domain twice, i believe it's worth writing a small functor, rather than copy pasting it